src/shell/js/composeui-fdc3/src/infrastructure/messages/Fdc3GetCurrentContextRequest.ts

```diff
@@ -0,0 +1,20 @@
+/* 
+ *  Morgan Stanley makes this available to you under the Apache License,
+ *  Version 2.0 (the "License"). You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0.
+ *  See the NOTICE file distributed with this work for additional information
+ *  regarding copyright ownership. Unless required by applicable law or agreed
+ *  to in writing, software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ *  or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ *  
+ */
+
+export class Fdc3GetCurrentContextRequest {
+    public readonly contextType?: string | null;
```

Comment:
Does it make sense to have both `undefined` and `null` as context type? It could be just `contextType?: string`.